mantidproject/mantid
# Autoscale and tile plots in muon analysis

**Description of work.**
The autoscale option was being reset to false when a tiled plot was selected and the user then edited on of the groups/pairs

-->

**To test:**
Open muon analysis
Load MUSR 62260
Turn on tiled plot
Turn on autoscale
Go to the grouping tab
Change the detectors for one of the groups
See that the autoscale box does not reset


Fixes #31478. 






---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
